packages/fontaine/src/cli.ts

```
@@ -0,0 +1,88 @@
+#!/usr/bin/env node
+import fs from 'node:fs/promises'
+import { resolve } from 'pathe'
+import { FontaineTransform } from './transform'
+
+async function main() {
+  const args = process.argv.slice(2)
```

ESLint

[error] Unexpected use of the global variable 'process'. Use 'require("process")' instead.

(node/prefer-global/process)